### PR TITLE
Pract 03: Add 04-05 | Militsa Toncheva

### PR DESCRIPTION
**Pull request by milicavilica**
Militsa Toncheva
Group 6 2MI0600216

### CODE REVIEW

File: SI/Sem.03/Pract.03/Task04.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	cin >> m >> n;
+	if (m <= 0 || n <= 0)
+	{
+		cout << "m and n must be larger than 0";
+	}
+	else
+	{
+		int p = m * n;
+
+		for (int i = 0; i < p; i++)
```
Comment (Justsvetoslavov) [inline SI/Sem.03/Pract.03/Task04.cpp:17]: we can start from 1 and condition will change to <=.
In addition, p is not necessary.

File: SI/Sem.03/Pract.03/Task04.cpp [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+		{
+			if (i % n == 0)
+			{
+				cout << endl;
+			}
+			cout << i + 1 << ' ';
```
Comment (Justsvetoslavov) [inline SI/Sem.03/Pract.03/Task04.cpp:23]: From the above changes , we will swap the if and the result and remove the +1 on the i

File: SI/Sem.03/Pract.03/Task05.cpp [hunk 3]
```diff
@@ -0,0 +1,41 @@
+#include <iostream>
+
+using namespace std;
+
+int main()
+{
+	int num = 0;
+	cout << "Enter a number: ";
+	cin >> num;
+	if (num < 0 || num > 100)
```
Comment (Justsvetoslavov) [inline SI/Sem.03/Pract.03/Task05.cpp:10]: Same as above.
Will return 1 and will have only the if statement.

File: SI/Sem.03/Pract.03/Task05.cpp [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+	{
+		cout << "the number must be between 0 and 100";
+	}
+	else
+	{
+		int counter = 0;
+		while (true)
```
Comment (Justsvetoslavov) [inline SI/Sem.03/Pract.03/Task05.cpp:17]: Re-think to remove the while (true).
Hint: use do-while.

**Review by Justsvetoslavov — CHANGES_REQUESTED** [review]

**Comment by GeorgiTerziev02** [comment]
The correct solutions have already been uploaded
https://github.com/Justsvetoslavov/Introduction_to_programming/tree/master/SI/Sem.03/Pract.03

**Pull request closed without merging**